### PR TITLE
Added the documentation for prefix mode on Windows

### PR DESCRIPTION
**Pull request by rawahars**
## Summary
With the launch of prefix delegation feature for Windows nodes in EKS, customers can now configure their Windows nodes to use IPv4 prefixes instead of secondary IPv4 addresses. This change adds the documentation about the best practises to be followed while enabling the feature for Windows nodes in EKS.

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/networking/prefix-mode/index_windows.md
Comment (sheetaljoshi) [inline content/networking/prefix-mode/index_windows.md:1]: Fix the grammar "In Amazon EKS, each pod is allocated a VPC-routable IP address, and therefore, the number of pods on the node is constrained by the available IP addresses even if there are sufficient resources to run more pods on the node. The IP address management for Windows nodes is performed by the [VPC Resource Controller,](https://github.com/aws/amazon-vpc-resource-controller-k8s) which runs in the control plane. More details about the workflow for IP address management on Windows nodes can be found [here](https://github.com/aws/amazon-vpc-resource-controller-k8s#windows-ipv4-address-management)."

Comment (sheetaljoshi) [inline content/networking/prefix-mode/index_windows.md:1]: Recommendation "Run all Pods on a node in the same mode" is misleading, can we move  it  under migration?

Comment (rawahars) [reply content/networking/prefix-mode/index_windows.md:1]: Yes sure. How does the following sound like-


> In Amazon EKS, each pod is allocated a VPC-routable IP address. Therefore, the number of pods on the node is constrained by the available IP addresses, even if there are sufficient resources to run more pods on the node. The IP address management for Windows nodes is performed by the VPC Resource Controller, which runs in the control plane. More details about the workflow for IP address management on Windows nodes can be found here.

Comment (rawahars) [reply content/networking/prefix-mode/index_windows.md:1]: Yes sure. Let me move it as a sub-section for migration.

**Pull request merged**